_includes/head.html

```diff
@@ -1,14 +1,32 @@
 <head>
 	<meta charset="UTF-8">
 	<title>{% if page.title %}{{ page.title | escape }}{% else %}{{ site.title | escape }}{% endif %}</title>
-	<meta name="viewport" content="width=device-width, initial-scale=1">    
+	<meta name="viewport" content="width=device-width, initial-scale=1">
 
-	<link type="text/css" rel="stylesheet" href="styles/styles.css">
-	<link type="text/css" rel="stylesheet" href="styles/font-awesome.min.css">
+	{% if site.github_hosted %}
+	<link type="text/css" rel="stylesheet" href="{{ '/styles/preloader.css?v=' | append: site.github.build_revision | relative_url }}">
+	<link type="text/css" rel="stylesheet" href="{{ '/styles/styles.css?v=' | append: site.github.build_revision | relative_url }}">
+	<link type="text/css" rel="stylesheet" href="{{ '/styles/font-awesome.min.css?v=' | append: site.github.build_revision | relative_url }}">
```

Comment:
I learned something new: 'relative_url' is new to me! Thanks... 

Two concerns which belong together:
1. You duplicate the code for the included files. Could you move this to "_includes/scripts.html"?
2. I very much wonder if this code would also work in the footer.

What do you think?

Reply:
Thank you, I will move it to the `scripts.html` file.

But I think, that stylesheets can't be implemented in the footer, but scripts can.

Thank you!

Reply:
Note for me: Fixed!